test/tap/enconding-tar-ball-for-scoped-packages.js

```diff
@@ -0,0 +1,81 @@
+'use strict'
+var test = require('tap').test
+var npm = require('../../lib/npm')
+var stream = require('readable-stream')
+
+var moduleName = 'xyzzy-wibble'
+var testModule = {
+  name: moduleName,
+  'dist-tags': {
+    latest: '1.3.0'
+  },
+  versions: {
+    '1.3.0': {
+      name: moduleName,
+      version: '1.3.0',
+      dist: {
+        shasum: 'da39a3ee5e6b4b0d3255bfef95601890afd80709',
+        tarball: 'https://registry.npmjs.org/xyzzy-wibble/-/xyzzy-wibble-1.3.0.tgz'
+      }
+    }
+  }
+}
+
+var scopedModuleName = '@scope/xyzzy-wibble'
+var scopedTestModule = {
+  name: scopedModuleName,
+  'dist-tags': {
+    latest: '1.8.0'
+  },
+  versions: {
+    '1.8.0': {
+      name: scopedModuleName,
+      version: '1.8.0',
+      dist: {
+        shasum: 'da39a3ee5e6b4b0d3255bfef95601890afd80709',
+        tarball: 'https://registry.npmjs.org/@scope/xyzzy-wibble/-/@scope/xyzzy-wibble-1.8.0.tgz'
+      }
+    }
+  }
+}
+
+var lastTarball
+
+test('setup', function (t) {
+  npm.load(function (a, b) {
```

Comment:
Hi

Thanks so much for the feedback. It is done.

Hugo

On Fri, Jan 22, 2016 at 4:59 PM, Rebecca Turner <notifications@github.com>
wrote:

> In test/tap/enconding-tar-ball-for-scoped-packages.js
> <https://github.com/npm/npm/pull/10947#discussion_r50611040>:
>
> > +  versions: {
> > +    '1.8.0': {
> > +      name: scopedModuleName,
> > +      version: '1.8.0',
> > +      dist: {
> > +        shasum: 'da39a3ee5e6b4b0d3255bfef95601890afd80709',
> > +        tarball: 'https://registry.npmjs.org/@scope/xyzzy-wibble/-/@scope/xyzzy-wibble-1.8.0.tgz'
> > +      }
> > +    }
> > +  }
> > +}
> > +
> > +var lastTarball
> > +
> > +test('setup', function (t) {
> > +  npm.load(function (a, b) {
>
> This and the next line can be:
>
> npm.load({loglevel: 'silly'}, function (a, b) {
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/npm/npm/pull/10947/files#r50611040>.
>